### DISCUSSION TITLE
Delivery: Uncertainty for non-analysts

### DISCUSSION THREAD

**Discussion by irisoren-sg** [best-practice-and-impact/aqua_book_revision#55] [General]
There is an unresolved comment in the word doc that probably does not constitute an issue. But noting here to consider how we want to address.

The text that it refers to is 

Government has produced a range of guidance to support analysts in presenting and communicating their analysis. This includes:

- The Office for Statistical Regulation’s [Approaches to presenting uncertainty in the statistical system](https://osr.statisticsauthority.gov.uk/publication/approaches-to-presenting-uncertainty-in-the-statistical-system/);
- The [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html);
- The Government Analysis Function guidance note [Communicating quality, uncertainty and change](https://analysisfunction.civilservice.gov.uk/policy-store/communicating-quality-uncertainty-and-change/);

The comment is:

```
There is a lot of material in here that is both accessible and useful for non-analysts. If we can’t/shouldn’t restructure and rebrand, then maybe a parallel ‘non-analyst’ version could be produced based almost entirely on the material already there.

15 February 2024 at 15:08
Friedman Andrew DWP FG CONTRACT MANAGEMENT AND PARTNER DELIVERY
I see your point and this could be where analysis meets wider competencies. Though depending on where the analysis needs to be used its all essential stuff. 
```

**Comment by rebecca-wodcke-gad** [comment]
Could we just add a sentence here that says something along the lines of "Although these were written with analysts in mind, they will be very useful for anyone involved in analysis". Doesn't feel beneficial to completely rewrite for non-analysts
